src/Card.js

```diff
@@ -0,0 +1,313 @@
+import Phaser from 'phaser'
+import { boardDrag, cardDimensions, hoverButtonRadius, cardBackFrame, cardDepth, activeDepth, hoverOffset, canvasHeight, inHandAdjustment, canvasWidth, inHandRange } from './Constants'
+export default class Card extends Phaser.GameObjects.Container {
+  constructor(scene, x, y, physicsGroup, cardNumber, orientation = Math.PI/2) {
+    super(scene, x, y)
+    //add images to container
+    this.shadow = scene.add.image(0,0,'shadow')
+    this.card = scene.add.sprite(0,0,'cardSprite')
+    this.card.setFrame(cardBackFrame)
+    this.flipButton = scene.add.image(cardDimensions.width/2 - hoverButtonRadius + hoverOffset, hoverButtonRadius - cardDimensions.height/2 - hoverOffset,'flip').setVisible(false)
+    this.rotateButton = scene.add.image(hoverButtonRadius - cardDimensions.width/2 + hoverOffset, cardDimensions.height/2 - hoverButtonRadius - hoverOffset,'rotate').setVisible(false)
+    this.shuffleButton = scene.add.image(cardDimensions.width/2 - hoverButtonRadius + hoverOffset, cardDimensions.height/2 - hoverButtonRadius - hoverOffset,'shuffle').setVisible(false)
+    this.stackCounter = scene.add.image(hoverButtonRadius - cardDimensions.width/2, hoverButtonRadius - cardDimensions.height/2,'deckCount').setVisible(true)
+    // this.count = scene.add.text(hoverButtonRadius - cardDimensions.width/2, hoverButtonRadius - cardDimensions.height/2, Number(10), { fontFamily: 'Georgia, "Goudy Bookletter 1911", Times, serif', color: 'rgb(0, 0, 0)' }).setVisible(true)
+    this.add([this.shadow,this.card,this.flipButton,this.rotateButton,this.shuffleButton,this.stackCounter])
+
+    //socket and room info for emit events
+    this.gameState = scene.game.gameState;
+    this.playerNumber = scene.game.playerNumber;
+    this.socket = scene.game.socket;
+
+    //set container size and add to scene, make interactive
```

Comment:
Stylistic comment - definitely not necessary - I love these comments, though since you've already identified your usable chunks of code it might be worth breaking them out into their own functions. Unless you're reusing this code elsewhere it might not save you "lines" of code but it'll make things semantically cleaner and'll make this function as a whole easier to parse.